docs/why-use-delta-lake.md

```diff
@@ -0,0 +1,96 @@
+
+Many basic data operations are hard in data lakes but quite easy with Delta Lake.  The only data operation that’s easy with in data lake is appending data.  Delta Lake makes all data operations easy including the following:
+
+* Appends
+* Upserts
+* Deletes
+* Replace where
+
+Even deleting a few rows of data from a data lake is hard.  It’s even harder if you want to run the operation in a performant manner.
+
+Delta Lake makes it easy to run common data operations and executes them performantly under the hood.
+
+Delta Lake also executes write operations as transactions, which makes data operations safer and prevents downtime.  Write operations will cause data lakes to be in an unstable state while the computations is running.  For example, if you read a data lake while a delete operation is running, then you may get the wrong data.
+
+Let’s explore the benefits of reliable transactions in more detail.
+
+## Reliable transactions
+
+Delta Lake supports transactions which means that write operations have the following characteristics:
+
+* They either finish completely or don’t run at all
+* They are executed in a serial manner and don’t conflict with other transactions
+* They don’t corrupt a table or violate table constraints
+
+Data lakes don’t support transactions, so the write operations are dangerous:
+
+* You can append data to a data lake with a mismatching schema
+* Your data lake often in an incorrect state while write transactions are performed
+* Data lakes can be corrupted for a variety of reasons
+* Concurrent transactions can cause data loss
+
+Production data systems should rely on storage systems like Delta Lake that support transactions.
+
+## Interoperability
+
+Delta Lake tables are interoperable and can be read/written by multiple different query engines.
+
+For example, you can create a Delta table with Spark, append to it with pandas, and then read it with Polars.
+
+![](delta-interop.png)
+
+Delta tables are powerful because they are interoperable with various query engines and computation runtimes.
+
+Suppose you have a Delta table that’s updated with an AWS Lambda function every 5 minutes.  There is only a small amount of data collected every 5 minutes, so a lightweight runtime like AWS Lambda is sufficient.
+
+Further suppose that the overall table is quite large.  So when you want to perform DML operations or query the whole table, your team uses a Spark cluster.
+
+Delta Lake is flexible to allow these types of operations from multiple readers and writers.  This provides teams with the flexibility to choose the right tool for the job.
+
+## Support for many languages
+
+Delta tables can be queried with a variety of different languages.  This project provides APIs for Rust and Python users and does not depend on Java or Scala.  This project is a great alternative for users like that Rust, pandas, Polars, DuckDB, or DataFusion.
+
+The Delta Spark connector has APIs in Java, Scala, Python, and R.
```

Comment:
maybe it can be confusing here, why we are talking about Delta Spark connector ?

Reply:
Yea, good call, removed this.